jobs/ci-kubernetes-e2e-gce-federation.env

```diff
@@ -8,7 +8,7 @@ KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_GCS_DEV_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_NODE_OS_DISTRIBUTION=debian
 
-GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\]
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]
```

Comment:
I see your point. WithCluster and NoCluster tests aren't compatible to be run in the same jobs. But what I am not sure about are other labels, particularly `Slow`, `Disruptive` and `Flaky`. We don't even have `Disruptive` and `Flaky` tests today. Should we proactively disable them? Also, `Slow` test is just slow but it can run in parallel. So we should probably just let them run in the jobs that don't necessarily have to be fast. More runs the better.

In my opinion, the only ones we should skip from all the non-main jobs (I am calling the regular GCE job as the main job) for now are `NoCluster` and `Serial` ones. Main job can skip all these and we can put them in the other job we are going to create for the serial ones anyway.